src/Utilities/BoGLServerRequest.test.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * BoGLServerRequest.test.js
+ * Testing for BoGL server requests
+ */
+
+import * as bogl from './BoGLServerRequest'
+
+const mockBoglAPI = {
+
+  // mock response for all endpoints
+  mockResponse: {
+    tag: "Log",
+    contents: "Mock BoGL Passed"
+  },
+  mockRequest: (url,data) => mockBoglAPI.mockResponse
+};
+
+
+// API Mocks
+it('AJAX Mock test', () => {
```

Comment:
This is a bit over my head

Reply:
Oh just some mocks. I.e. instead of running against an actual server (which has it's own tests), I'm just testing the piping of the frontend. Basically if I control how the response is made, and what the response is, does it work as expected. This is helpful to check for handling related issues to inputs, but these tests are still a bit sparse currently.